lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -310,4 +310,10 @@ def self.ae_user_object(options = {})
       $miq_ae_logger.info("User [#{obj.userid}] with current group ID [#{obj.current_group.id}] name [#{obj.current_group.description}]")
     end
   end
+
+  V2_PASSWORD_REGEXP = /v([0-9]+)%3A%7B(.*?)%7D/
```

Comment:
Push this to the top of the file to group with the other constant.  Also, I would recommend adding a comment clarifying why this one is different then the one in `MiqPassword`.